synthesizer/program/src/traits/stack_and_registers.rs

```diff
@@ -242,11 +242,16 @@ pub fn types_equivalent<N: Network>(
             struct_compare(&*external_stack, struct_type0, stack1, struct_type1)
         }
         (Struct(id), ExternalStruct(loc)) => {
+            dbg!("HERE!");
```

Comment:
Now we know how the sausage is made.

Reply:
OH no... isn't `HERE` universal though? 😆